### PR TITLE
r/kubernetes_cluster: switching back to parallel tests

### PR DESCRIPTION
**Pull request by tombuildsstuff**
Trialled switching back to regular (non-parallel'd) test cases to work around API performance issues, but this ultimately wasn't fruitful

### CODE REVIEW

File: azurerm/internal/services/containers/resource_arm_kubernetes_cluster.go [hunk 1]
```diff
@@ -237,6 +237,28 @@ func resourceArmKubernetesCluster() *schema.Resource {
 				},
 			},
 
+			"service_principal": {
```
Comment (katbyte) [inline azurerm/internal/services/containers/resource_arm_kubernetes_cluster.go]: I think you'll want to revert this change?

Comment (tombuildsstuff) [reply azurerm/internal/services/containers/resource_arm_kubernetes_cluster.go]: nah this is just a minor regrouping from where the `service_principal` block went from required -> optional -> required in the last PR and I didn't want travis to block the release - so this should be fine 👍
